### ISSUE TITLE
Unknown Linux Button 319

### ISSUE THREAD

**Issue by zeddidragon** [libcala/stick#42]
**Describe the bug**
When pressing M1 on a Flydigi Apex 2, it's not recognized by stick and an error is output. Error message as following:

```
  Unknown Linux Button 319
  Report at https://github.com/libcala/stick/issues
```

**To Reproduce**

1. Use a Flydigi Apex 2. Other devices might work as well.
2. Press M1, located on the inside of the right grip.
3. Observe error message.

**Expected behavior**
Similar behavior as other buttons:
```
  p2: Number(15) Pushed
  p2: Number(15) Released
```

**Desktop:**
```
    ❯ lsb_release -a
    No LSB modules are available.
    Distributor ID: Pop
    Description:    Pop!_OS 21.04
    Release:        21.04
    Codename:       hirsute
```

**Additional context**
~~I assume Stick currently supports 16 buttons, numbered 0-15, although I haven't looked yet. I'm just reporting 'cause the program told me to.~~

Looking at the source code, I see where my button is supposed to go.
```
        0x130 /* BTN_A / BTN_SOUTH */ => Event::ActionA(pushed),
        0x131 /* BTN_B / BTN_EAST */ => Event::ActionB(pushed),
        0x132 /* BTN_C */ => Event::ActionC(pushed),
        0x133 /* BTN_X / BTN_NORTH */ => Event::ActionV(pushed),
        0x134 /* BTN_Y / BTN_WEST */ => Event::ActionH(pushed),
        0x135 /* BTN_Z */ => Event::ActionD(pushed),
        0x136 /* BTN_TL */ => Event::BumperL(pushed),
        0x137 /* BTN_TR */ => Event::BumperR(pushed),
        0x138 /* BTN_TL2 */ => Event::TriggerL(f64::from(u8::from(pushed)) * 255.0),
        0x139 /* BTN_TR2 */ => Event::TriggerR(f64::from(u8::from(pushed)) * 255.0),
        0x13A /* BTN_SELECT */ => Event::MenuL(pushed),
        0x13B /* BTN_START */ => Event::MenuR(pushed),
        0x13C /* BTN_MODE */ => Event::Exit(pushed),
        0x13D /* BTN_THUMBL */ => Event::Joy(pushed),
        0x13E /* BTN_THUMBR */ => Event::Cam(pushed),
        // <-- Right here, 0x13F

        0x220 /* BTN_DPAD_UP */ => Event::Up(pushed),
        0x221 /* BTN_DPAD_DOWN */ => Event::Down(pushed),
 	0x222 /* BTN_DPAD_LEFT */ => Event::Left(pushed),
 	0x223 /* BTN_DPAD_RIGHT */ => Event::Right(pushed),
```
`0x13F`, along with all the other specific named buttons. I'd add it myself but I don't understand the naming scheme used. What makes X and Y into V and H? The other buttons underneath my pad show up as numbers 11-13 so they're no help as a reference. Obviously adhering to a specific chinese novelty controller isn't of the highest priority, but given it's intended position in the code this button `0x13F` needs a name and I don't know what.

**Comment by zeddidragon** [comment]
https://github.com/libcala/stick/pull/43

**Comment by AldaronLau** [comment]
@zeddidragon Thanks for the issue and PR!

#### About the naming scheme used by stick:
There's two kinds of naming schemes used for controllers usually: A/B/X/Y (or shapes), and North/South/East/West.  Stick's naming scheme is based on the fact that buttons A and B tend to have meanings associated with their name (Accept and Cancel usually), while X and Y tend to have meanings associated with their position on the controller.  So V is vertical (the higher of the buttons) and H is horizontal (the lower of the two buttons).

BTW one of the goals of stick is to support obscure controllers that libraries based on the w3 standard gamepad model such as the gilrs crate can not.

**Comment by AldaronLau** [comment]
@zeddidragon Your PR has been merged and released in stick version 0.12.3 on crates.io, so I believe that resolves this issue (re-open if you disagree).  Closing now.
